Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md

```diff
@@ -0,0 +1,142 @@
+# RESPONSIVENESS IN CSS
+
+ As we all know webpages can be viewed on many devices like desktops, tablets, and phones. For web pages to look attractive and eye-catching on each device , responsiveness should be used . Without this our website may look interactive in some dimensions or on some devices like laptops but may look undesirable and unorganized in other devices like phones, tablets etc.
+
+> We  use CSS and HTML to resize, hide, shrink, enlarge, or move the content to make it look good on any screen. 
+## USING MEDIA QUERIES
+
+It uses the @media rule to include a block of CSS properties only if a certain condition is true.
+
+**Media queries can be used to change orientation the browser as shown in given example.**
+
+1)@media screen and [media feature rules]
+
+```html
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<style>
+body {
+  background-color: lightgreen;
+}
+@media only screen and (orientation: landscape) {
+  body {
+    background-color: lightblue;
+  }
+}
+</style>
+</head>
+<body>
+
+<p>Resize the browser window. In landscape its background color is "lightblue", otherwise it is "lightgreen".</p>
+
+</body>
+</html>
+
+```

```

Comment:
Wrong image captions and example taken from https://www.w3schools.com/css/tryit.asp?filename=tryresponsive_mediaquery_orientation2

Please try to do examples on your own, or else at least specify the source.